### ISSUE TITLE
Rentable Room: Chest and Tome Items place at bad angles and can't be repositioned

### ISSUE THREAD

**Issue by jrotunda** [Meridian59/Meridian59#646]
For the personal Rentable Room from the various Inns (currently Brownestone Inn, Limping Toad and Aerie Guest House), users can place decorative items.

Many of these items have one viewing angle - a candelabra, an orc torch, a stool, an urn - but some of them have specific angles to them, that when placed by a user, may not be a good angle.

Two of these items are "tome" (jbook_icon_rsc) and the "personal chest" (smallbox.kod).

If you place a tome in a corner, it may be at an angle undesired, same with the box. Both of these items will default to the "East" direction when placed. 

To fix this, a "change direction" command should be added to the room decorator for these two items, and when any ornamental object or item is added to the rentable room in the future. The decorator already has a lot of refined tools at its disposal, and this would be something a user would expect it to have.
![M59 - RentableRoom - Personal Chest - Wall Facing 2](https://github.com/Meridian59/Meridian59/assets/2608213/6acf4751-499f-4892-bf33-acbbfa70a513)
![M59 - RentableRoom - Personal Chest - Wall Facing](https://github.com/Meridian59/Meridian59/assets/2608213/4cfe426f-1167-4079-af72-c5dd282b2f14)

**Comment by davidklaw** [comment]
I think this might be complicated by the fact that multiple instances of the same object can be placed in a room. I thought that maybe all instances of a object type could be rotated but that's undesirable. A simple solution we could implement quickly is placing the item at the angle the player is facing. However, since the chest and tome can't be refunded into heartstones, a bad angle is a costly mistake. Maybe that's a good next step though?

💡 It would be neat if object dialogs had controls for translating and rotating objects in rooms, allowing for finer control over individual decorations.

**Comment by jrotunda** [comment]
You're right it can't be flipping all of the same objects and that presents a dilemma - same with how removing an object, once placed, wouldn't work individually - a player would have to remove all of that object type from the room (I was going to write an issue up that items can't be removed once placed but it seems like a headache to code)

Having it face the direction of the player would at least let the player know what direction it will face upon placement. 

The decorator should also mention that the item would be facing them so that's a dialogue edit.

Right now, since the rooms are instanced creations, they don't know what angle of the room they're in.

**Comment by jrotunda** [comment]
You're right it can't be flipping all of the same objects and that presents a dilemma - same with how removing an object, once placed, wouldn't work individually - a player would have to remove all of that object type from the room (I was going to write an issue up that items can't be removed once placed but it seems like a headache to code)

Having it face the direction of the player would at least let the player know what direction it will face upon placement. 

The decorator should also mention that the item would be facing them so that's a dialogue edit.

Right now, since the rooms are instanced creations, they don't know what angle of the room they're in.
